frontend/public/components/build.tsx

```diff
@@ -107,7 +108,7 @@ export const BuildEnvironmentComponent = (props) => {
   </div>;
 };
 
-const pages = [navFactory.details(BuildsDetails), navFactory.editYaml(), navFactory.envEditor(BuildEnvironmentComponent), navFactory.logs(BuildLogs)];
+const pages = [navFactory.details(BuildsDetails), navFactory.editYaml(), navFactory.envEditor(BuildEnvironmentComponent), navFactory.logs(BuildLogs), navFactory.events(ResourceEventStream)];
```

Comment:
nit:

```JavaScript
const pages = [
  navFactory.details(BuildsDetails), 
  navFactory.editYaml(), 
  navFactory.envEditor(BuildEnvironmentComponent), 
  navFactory.logs(BuildLogs), 
  navFactory.events(ResourceEventStream)
];
```
There are enough items in this array that it is rather difficult to read.